lib/ramble/ramble/cmd/results.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import llnl.util.tty as tty
+
+from ramble.config import ConfigError
+import ramble.experimental.uploader
+
+
+description = "take actions on experiment results"
+section = "workspaces"
+level = "short"
+
+subcommands = [
+    'file_upload',
+]
+
+#: Dictionary mapping subcommand names and aliases to functions
+subcommand_functions = {}
+
+
+def results_file_upload_setup_parser(subparser):
+    subparser.add_argument(
+        'filename', metavar='filename',
```

Comment:
Do we also want to support a short arg like `-f`?

Reply:
Since filename is the only argument for upload, it doesn't take a flag. Syntax is 'ramble results upload <filename>'